### ISSUE TITLE
hello，你们训练好模型在哪啊

### ISSUE THREAD

**Issue by cumthxy** [EuphoriaYan/zi2zi-pytorch#2]
n our pre-trained model, the mapping relationships between label and writers are below:

**Comment by EuphoriaYan** [comment]
抱歉暂时还不能放出（因为目前工作要用到），大约会在9/15之后放出，谢谢qwq

**Comment by Crescentz** [comment]
请问你们运行 python package.py 没有遇到错误么？
`with open(dst_json, 'r', encoding='utf-8') as fp:
        dst_fonts = json.load(fp)`
没有找到这个字典的任何描述啊

**Comment by EuphoriaYan** [comment]
突然想到还没放模型hhh 已经放上去了，抱歉hhh
